docs/ABOUT.md

```diff
@@ -0,0 +1,11 @@
+# What is JavaScript?
```

Comment:
This shows up on a page that already has a title, so it's cleaner if this is just paragraph text.